templates/vue/src/components/TapestryMap.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div>
+    <div style="height: 800px; width: 100%">
+      <l-map
+        :options="mapOptions"
+        :bounds="setBounds"
+        :max-bounds="worldBounds"
+        style="height: 80%"
+        @update:center="updateCenter"
+        @update:zoom="updateZoom"
+      >
+        <l-tile-layer :url="url" :attribution="attribution" />
+        <l-marker
+          v-for="marker in markerlocations"
+          :key="marker.id"
+          :lat-lng="marker.pos"
+          :icon="marker.access ? icon : inaccessibleIcon"
+        >
+          <l-popup>
+            <div v-if="marker.access" @click="openNode(marker.id)">
+              {{ marker.title }}
```

Comment:
Can you add a style for this and set `cursor: pointer;` so it appears more like a link? Maybe also make it bold. (This does not apply for the inaccessible nodes)